src/networks.json

```diff
@@ -875,6 +890,23 @@
     "start": 53370,
     "imageIPFS": "QmPXPCBho3kGLt5rhG9JGkKmzdtLvqZmJqGzzijVCuggWY"
   },
+  "5851": {
+    "key": "5851",
+    "name": "Ontology Testnet",
+    "chainId": 5851,
+    "network": "testnet",
+    "multicall": "0x381445710b5e73d34aF196c53A3D5cDa58EDBf7A",
```

Comment:
I can't check the multicall code here https://explorer.ont.io/testnet/contract/other/0x381445710b5e73d34aF196c53A3D5cDa58EDBf7A/10/1 

Is there a way to check the source code of the contract?

Reply:
@lucas7788 Can you plz check this too plz

Reply:
we have fixed this problem, please tale a look again.